apps/masterbots.ai/lib/hooks/use-thread.tsx

```
@@ -113,85 +118,85 @@ export function ThreadProvider({ children }: ThreadProviderProps) {
-        )}].  Then answer this question: ${fullMessage}`
-      })
+          allMessages,
+        )}].  Then answer this question: ${fullMessage}`,
+      });
     },
-    [activeThread?.threadId, allMessages, append, session]
-  )
+    [activeThread?.threadId, allMessages, append, session],
+  );
 
   React.useEffect(() => {
```

Biome

[error] This hook does not specify all of its dependencies: fetchMessages

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)